src/arrayMethodSort.js

```diff
@@ -4,8 +4,32 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = initialCompare) {
+    let count = 1;
+
+    while (count) {
+      count = 0;
+
+      for (let i = 0; i < this.length - 1; i++) {
+        if (compareFunction(this[i], this[i + 1]) > 0) {
```

Comment:
```suggestion
        if (compareFunction(this[i], this[i + 1]) > 0) {
```
For better readability it would be better to create variables for `this[i]` and `this[i + 1]`. Like `next` and `current`
```